### PR TITLE
MeshPart: fix header uniformity + remove superfluous whitespace

### PR DESCRIPTION
**Pull request by luzpaz**
Adding some uniformity to the headers of the `src/Mod/MeshPart`. I'm not sure if the `blame` info for some of the files is accurate since the source code wasn't originally on github and some of that history is not available.

### CODE REVIEW

File: src/Mod/MeshPart/InitGui.py [hunk 1]
```diff
@@ -26,44 +26,46 @@
...
 
-	def Initialize(self):
-		# load the module
-		import MeshPartGui
-		import MeshPart
-	def GetClassName(self):
-		return "MeshPartGui::Workbench"
 
-#Gui.addWorkbench(MeshPartWorkbench())
+def Initialize(self):
```
Comment (chennes) [inline src/Mod/MeshPart/InitGui.py:62]: It looks like you've lost an indentation level here.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5135">pipeline status</a>for feature branch [PR_5135](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5135). Pipeline [#399136857 ](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/399136857) was triggered at [facbebb](https://github.com/FreeCAD/FreeCAD/pull/5135/commits/facbebba53810a61403376769d167e197982d692). All CI branch [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/MeshPart/InitGui.py [hunk 2]
```diff
@@ -26,44 +26,46 @@
...
+    # load the module
+    import MeshPartGui
+    import MeshPart
+
+
+def GetClassName(self):
+    return "MeshPartGui::Workbench"
```
Comment (chennes) [inline src/Mod/MeshPart/InitGui.py:69]:
```suggestion
    def Initialize(self):
        # load the module
        import MeshPartGui
        import MeshPart


    def GetClassName(self):
        return "MeshPartGui::Workbench"
```

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5135">pipeline status</a> for feature branch [PR_5135](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5135). Pipeline [400395832](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/400395832) was triggered at [9e82952](https://github.com/FreeCAD/FreeCAD/pull/5135/commits/9e82952296f7d33ce75b6f89af9b031e69434185). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
The PR looks good but the CI failed for an unknown reason as far as I can see. Could you please merge the 3 commits into one to trigger the CI again? Just to be on the safe.

**Comment by donovaly** [comment]
Interesting that the CI made a re-run. However, it succeeded and it is merged now.

**Pull request merged**